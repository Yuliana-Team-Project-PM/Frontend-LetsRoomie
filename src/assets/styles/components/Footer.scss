@import '../Vars.scss';

.footer{
    color: white;
    background-color:$dark-color;

    &__cta {
        height: 110px;
        text-align: center;
        padding: 10px;
        border-bottom: .5px solid $dark-color;

        h3 {
            margin: 1em 0 0 0;
        }

        &--button {
            @include base-button($gray-color, $secondary-color, $border-button);
        }
    }

    &__icons {
        display: flex;
        justify-content: space-between;
        align-items: center;
        // margin: 0px 100px;
    }
}

@media screen and (max-width: 426px) {
    .footer {
        // $container(400px);

        &__icons {
            display: block;

            &--social {
                // display: flex;
                text-align: center;
                margin: 3em 0 ;
            }

            &--copy {
                text-align: center;
                // margin: .6em 0;
                padding-bottom: 1em;

                p {
                    margin: 0;
                }
            }
        }
    }
}

@media screen and (max-width: 321px) {
    .footer {
        &__cta {
            h3 {
                font-size: .85em;
            }
        }
    }
}

