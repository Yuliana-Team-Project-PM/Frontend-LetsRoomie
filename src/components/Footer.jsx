import React from 'react';
import { Link } from 'react-router-dom';

import '../assets/styles/components/Footer.scss';

const Footer = () => {
    return(
        <footer className="footer">
            <section className="footer__cta">
                <h3>Miles de personas han encontrado su Roomate apropiado, ahora es tu turno</h3>
                <Link to='select-acount'><button className="footer__cta--button">LET'S ROOMIE</button></Link>
            </section>
            <section className="footer__icons">
                <div>
                    <p>Â© 2020 Lets Rommie Technologies, Inc.</p>
                </div>
                <div>
                    <p>Nuestras redes sociales</p>
                    <a href="#"><img alt="icono de FaceBook" src="https://img.icons8.com/color/48/000000/facebook.png"/></a>
                    <a href="#"><img alt="icono de Instagram" src="https://img.icons8.com/fluent/48/000000/instagram-new.png"/></a>
                    <a href="#"><img alt="icono de Twitter" src="https://img.icons8.com/fluent/48/000000/twitter.png"/></a>
                    <a href="#"><img alt="icono de whatsapp" src="https://img.icons8.com/color/48/000000/whatsapp.png"/></a>
                </div>
            </section>
        </footer>
    );
};

export default Footer;