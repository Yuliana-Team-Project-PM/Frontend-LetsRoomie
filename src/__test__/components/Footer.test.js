import React from 'react';
import { mount } from 'enzyme';
import { MemoryRouter } from 'react-router-dom';
import Footer from '../../components/Footer';

describe('<Footer />', () => {
    const footer = mount(
        <MemoryRouter>
            <Footer />
        </MemoryRouter>
    );
    
    test('Render del componente Footer', () => {
        expect(footer.length).toEqual(1);
    });

    test('Footer tiene que tener 1 Link', () => {
        expect(footer.find('Link')).toHaveLength(1);
    })

    test('Footer tiene que tener 5 anchors', () => {
        expect(footer.find('a')).toHaveLength(5);
    })
});