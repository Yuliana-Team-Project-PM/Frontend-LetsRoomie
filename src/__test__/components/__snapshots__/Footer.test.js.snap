// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> Footer Snapshot 1`] = `
<footer
  className="footer"
>
  <section
    className="footer__cta"
  >
    <h3>
      Miles de personas han encontrado su Roomate apropiado, ahora es tu turno
    </h3>
    <a
      href="/select-acount"
      onClick={[Function]}
    >
      <button
        className="footer__cta--button"
      >
        LET'S ROOMIE
      </button>
    </a>
  </section>
  <section
    className="footer__icons"
  >
    <div>
      <p>
        Â© 2020 Lets Rommie Technologies, Inc.
      </p>
    </div>
    <div>
      <p>
        Nuestras redes sociales
      </p>
      <a
        href="#"
      >
        <img
          alt="icono de FaceBook"
          src="https://img.icons8.com/color/48/000000/facebook.png"
        />
      </a>
      <a
        href="#"
      >
        <img
          alt="icono de Instagram"
          src="https://img.icons8.com/fluent/48/000000/instagram-new.png"
        />
      </a>
      <a
        href="#"
      >
        <img
          alt="icono de Twitter"
          src="https://img.icons8.com/fluent/48/000000/twitter.png"
        />
      </a>
      <a
        href="#"
      >
        <img
          alt="icono de whatsapp"
          src="https://img.icons8.com/color/48/000000/whatsapp.png"
        />
      </a>
    </div>
  </section>
</footer>
`;
